TrackingTools/Producers/python/trajectoryCleanerESProducer_cfi.py

```diff
@@ -0,0 +1,11 @@
+import FWCore.ParameterSet.Config as cms
+
+## TEMPLATE as it is used by HLT
+trajectoryCleanerBySharedHits = cms.ESProducer( "TrajectoryCleanerESProducer",
```

Comment:
what are we supposed to do with 
https://github.com/cms-sw/cmssw/blob/CMSSW_7_4_0_pre5/TrackingTools/TrajectoryCleaning/python/TrajectoryCleanerBySharedHits_cfi.py
``` trajectoryCleanerBySharedHits = cms.ESProducer("TrajectoryCleanerESProducer", ```
?
the same name, but in a different place and with different parameters.

Note that the style rules for cfi files is to define modules matching the name of the file (trajectoryCleanerESProducer_cfi vs trajectoryCleanerBySharedHits)